src/Core/src/Platform/Windows/NativeVersion.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Microsoft.UI.Xaml;
+
+namespace Microsoft.Maui
+{
+	public static class NativeVersion
+	{
+		public static bool IsDesktop =>
+			Window.Current == null;
```

Comment:
What does this actually check? Don't all windows apps have a Window?

Reply:
https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.window.current?view=winui-3.0

> The currently activated window for UWP apps. Null for Desktop apps.


Added a comment